packages/ui/src/common/components/Modal/Modal.tsx

```diff
@@ -211,6 +211,8 @@ export const ModalWrap = styled.section<ModalWrapProps>`
     switch (modalMaxSize) {
       case 'm':
       case 'l':
+        // HACK: Move modals to the top on mobile because SubWallet shows the keyboard on the viewport
+        // instead of reducing the viewport size (at least on Android).
```

Comment:
Is that hack already implemented and you're only adding a comment or is that a todo?

Reply:
It's already implemented in #4708. It's just that I realized that even I will probably have a hard time figuring out why I added these in a few weeks, so I added these 2 comments.